tests/github/interactive_api_director/experiments/pytorch_kvasir_unet/data_loader.py

```diff
@@ -2,16 +2,22 @@
 from skimage import io
 import numpy as np
 from openfl.utilities import validate_file_hash
+import requests
```

Comment:
Imports are not in right order. Probably after these changes is it possible to remove `tests/github/interactive_api_director` from excluded paths for flake8?

Reply:
Makes sense. I will create a separate pull request for this after merging this one.